sdk/detect/gce/gce.go

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Vendor specific API integration should be kept in the https://github.com/open-telemetry/opentelemetry-go-contrib repository.

Reply:
I agree. Does it apply to the interface of vendor specific API, auto.go?

Reply:
If OTEP-111 were to be accepted, I'd expect the `Detector` interface and global `Detect()` method to live in this repo, potentially with `Detector` implementations for vendor-neutral resources like k8s metadata or process info, while vendor-specific `Detector` implementations, such as those for GCE or AWS, would live in the opentelemetry-go-contrib repo.

Reply:
Thank you for the information. Could @james-bebbington  please confirm the case of OTEP-111? Thanks.